pkg/api/folder.go

```diff
@@ -917,19 +853,93 @@ func (fk8s *folderK8sHandler) newToFolderDto(c *contextmodel.ReqContext, item un
 		uid := parentsFullPathUIDs[1:][i]
 		url := dashboards.GetFolderURL(uid, slug)
 
-		parents = append(parents, dtos.Folder{
+		ff := folder.Folder{
 			UID:   uid,
-			OrgID: c.SignedInUser.GetOrgID(),
 			Title: v,
 			URL:   url,
-		})
+		}
+		parentDTO, err := fk8s.toDTO(c, &ff, "", checkCanView)
+		if err != nil {
+			// #TODO should we log this error?
```

Comment:
I think we should log the error here... the [(hs *HTTPServer) newToFolderDto](https://github.com/grafana/grafana/blob/927d0dddfe5a0b25ab5f8c35cd0aa3482d5d522a/pkg/api/folder.go#L442) is emitting a log message for this case

Reply:
Good point. We need to add logging overall. I will add this as a separate task.